index.html

```diff
@@ -39,13 +42,113 @@ <h1>Lofi girl</h1>
           </div>
          </div>
 
-         <div class="row" id="">
-
-          <div class="col"></div>
-
-         </div>
-
-         <!-- Footer-->
+         <!-- LIST -->
+         <div class="card" id="list">
+          <ul class="list-group list-group-flush">
+
+            <li class="list-group-item">
```

Comment:
Am aware this was noted, but for completion sake: still need DueDate and Status to be included properly